framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -487,14 +487,16 @@ private boolean isExpressionOrStatementPure(
    * @param classTree the class that contains {@code methodTree}
    * @param methodTree the method or constructor tree
    */
+  // TODO: should field visibility matter? An access from outside the class might observe
+  // the declared type instead of a refined type. Issue a warning to alert users?
   private void addInitialFieldValues(S store, ClassTree classTree, MethodTree methodTree) {
     boolean isConstructor = TreeUtils.isConstructor(methodTree);
     TypeElement classEle = TreeUtils.elementFromDeclaration(classTree);
     for (FieldInitialValue<V> fieldInitialValue : analysis.getFieldInitialValues()) {
       VariableElement varEle = fieldInitialValue.fieldDecl.getField();
       // Insert the value from the initializer of private final fields.
```

Comment:
This comment is now incorrect.  I would move the TODO that's on the method to here.